app/transcribe/run_tests.bat

```diff
@@ -0,0 +1,2 @@
+SET PYTHONPATH=%cd%;%cd%\..\..
+python -m unittest --verbose .\tests\test_audio_player.py
```

Comment:
Should be generic so as new test files are added do not need to modify this file

Reply:
👍